ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -550,7 +562,7 @@ export class AccountCreation extends React.Component<
                     inputName='givenName'
                     placeholder='First Name'
                     invalid={givenName.length > nameLength}
-                    labelText='First Name'
+                    labelText={FORM_LABELS.givenName}
```

Comment:
can also replace the placeholders with this

Reply:
Done 👍